app/controllers/users_controller.rb

```diff
@@ -185,6 +196,10 @@ def test_mail
 
   private
 
+  def mod_auth_openidc_session_exist?
```

Comment:
Why is this needed? Can you have oidc enabled but have sessions coming from another auth provider?

Reply:
while using `mod_auth_openidc`, it stores sessions in the cookie, and we need to logout from mod_auth_openidc as well. For this I had gone through two articles:
1) https://github.com/zmartzone/mod_auth_openidc/wiki/OpenID-Connect-Session-Management#logout
2) https://community.auth0.com/t/users-cant-logout-with-mod-auth-openidc/22125/3

I hope this answers your question.